packages/tc-ui/src/lib/mappers/get-value.js

```diff
@@ -0,0 +1,39 @@
+const sortBy = require('lodash.sortby');
+
+module.exports = (itemSchema, itemValue) => {
```

Comment:
similar comment to on previous file, but also add that if you use

```
const myFunc = // define the function

module.exports = { myFunc }

```

then error stack traces and debugger tools will have information about the function name, rather than anonymous function